src/core/Akka.Remote.Tests/Transport/DotNettyMutualTlsSpec.cs

```diff
@@ -0,0 +1,168 @@
+//-----------------------------------------------------------------------
+// <copyright file="DotNettyMutualTlsSpec.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2022 Lightbend Inc. <http://www.lightbend.com>
+//     Copyright (C) 2013-2025 .NET Foundation <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using Akka.Actor;
+using Akka.Configuration;
+using Akka.TestKit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Akka.Remote.Tests.Transport
+{
+    /// <summary>
+    /// Tests mutual TLS authentication enforcement in DotNetty transport.
+    /// When require-mutual-authentication is enabled, both client and server must
+    /// present valid certificates with accessible private keys.
+    /// </summary>
+    public class DotNettyMutualTlsSpec : AkkaSpec
```

Comment:
The default SSL / TLS specs already cover the happy path scenario - we should rewrite this to ensure hard fail if the two different, but valid certificates are used - and maybe another scenario where the same certificate is used but the password / private key auth is incorrect on one of the nodes.